### ISSUE TITLE
Percent threshold help text is wrong

### ISSUE THREAD

**Issue by phumpal** [atlassian/gostatsd#181]
gostatsd may not flush /calculate additional percentiles when `--percent-threshold {90,95,99}`

For example, w/systemd

```
[Unit]
Description=gostatsd
After=network.target

[Service]
Type=simple
Restart=on-failure
RestartSec=3s
WorkingDirectory=/home/gostatsd
ExecStart=/bin/bash -c "/home/gostatsd/gocode/bin/gostatsd \
  --config-path=/home/gostatsd/config/staging.toml \
  --percent-threshold {90,95,99} \
  --flush-interval 10s \
  --verbose"

[Install]
WantedBy=multi-user.target
```

`upper_90` exists but not `upper_95` or `upper_99`.

**Comment by tiedotguy** [comment]
I believe what is happening here is it can't parse ```{90,95,99}``` so it falls back to the default (```90```).  Arguably it should fail to start in this case.

Try using ```--percent-threshold "90 95 99"```.

**Comment by phumpal** [comment]
Thanks @tiedotguy! 

I updated `ExecStart` to

```
ExecStart=/home/gostatsd/gocode/bin/gostatsd \
  --config-path=/home/gostatsd/config/staging.toml \
  --percent-threshold "90 95 99"  \
  --flush-interval 10s \ 
  --verbose
```

which did the trick. 

Maybe a PR is in order to fix `-help` or I am misunderstanding?

```
--percent-threshold string                       Comma-separated list of percentiles (default "90")
```

**Comment by tiedotguy** [comment]
Nope, not a misunderstanding! It's legacy (buggy) text, from behavior changed in #130, I'll keep this issue to fix it.

**Comment by tiedotguy** [comment]
Addressed by #182

**Comment by tiedotguy** [comment]
This shook out another bug in viper, addressed in 7.0.1

**Comment by phumpal** [comment]
Thanks @tiedotguy
